model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -919,15 +931,23 @@ async def execute(
                             token=result["result"]["token"]["text"],
                             log_prob=result["result"]["token"]["logprob"],
                         )
-                    yield CompletionStreamV1Response(
-                        request_id=request_id,
-                        output=CompletionStreamOutput(
-                            text=result["result"]["token"]["text"],
-                            finished=finished,
-                            num_completion_tokens=num_completion_tokens,
-                            token=token,
-                        ),
-                    )
+                    try: 
+                        yield CompletionStreamV1Response(
+                            request_id=request_id,
+                            output=CompletionStreamOutput(
+                                text=result["result"]["token"]["text"],
+                                finished=finished,
+                                num_completion_tokens=num_completion_tokens,
+                                token=token,
+                            ),
+                        )
+                    except Exception as e:  # if result["result"]["token"]["text"] doesn't exist 
+                        if "error" in result:
+                            logger.exception(f"Error parsing text-generation-inference stream output. Error message: {e}")
+                            raise ObjectHasInvalidValueException(
+                                f"Error parsing text-generation-inference stream output." 
```

Comment:
i would expect same error processing logics here as in sync use case

Reply:
updated to reflect that now

Reply:
raise InvalidRequestException or UpstreamServiceError?